README.md

```diff
@@ -33,6 +33,10 @@ Then screw the top part and turn the board on!
 
 Now you can also program the board with the USB-C. I suggest to use ESPHOME beacuse it's super easy to setup and to integrate with Home Assistant
```

Comment:
```suggestion
Now you can also program the board with the USB-C. The community suggests you use ESPHome Web because it's super easy to setup and to integrate with Home Assistant.
```